HackerRank Problem/Python/DrawingBook.py

```diff
@@ -0,0 +1,34 @@
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the pageCount function below.
+#
+def pageCount(n, p):
+    #
+    # Write your code here.
+    #
+    
+    countFront = p//2
+    countBack = n//2 - p//2
+    
+    return min(countFront, countBack)
+
+
+    
+
+if __name__ == '__main__':
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    n = int(input())
+
+    p = int(input())
+
+    result = pageCount(n, p)
+
+    fptr.write(str(result) + '\n')
+
+    fptr.close()
+
```

Comment:
@camfilho please remove the extra lines after each statement.

Reply:
all the empty lines? @aneels3